resolwe_bio/tests/workflows/test_mirna.py

```diff
@@ -40,5 +53,5 @@ def test_mirna_workflow(self):
             workflow, "rc", "mirna_featurecounts_rc.tab.gz", compression="gzip"
         )
         self.assertFile(
-            workflow, "exp", "mirna_featurecounts_tpm.tab.gz", compression="gzip"
```

Comment:
The old file needs to be deleted. Also, run the test locally, end commit the file `mirna_featurecounts_cpm.tab.gz` created by the test.